### PR TITLE
[DA-1574] MIGRATION: Adding COPE survey fields to participant summary

### PR DESCRIPTION
**Pull request by kenny-skaggs**
Adds fields to the participant summary table for May, June, and July survey results.

### CODE REVIEW

File: rdr_service/alembic/versions/16452fdb1334_summary_cope_survey_fields.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_june_time', rdr_service.model.utils.UTCDateTime(), nullable=True))
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_may', rdr_service.model.utils.Enum(QuestionnaireStatus), nullable=True))
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_may_authored', rdr_service.model.utils.UTCDateTime(), nullable=True))
+    op.add_column('participant_summary', sa.Column('questionnaire_on_cope_may_time', rdr_service.model.utils.UTCDateTime(), nullable=True))
+    # ### end Alembic commands ###
+
+
+def downgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('resource_search_results', 'page_no',
```
Comment (j-kanuch) [inline rdr_service/alembic/versions/16452fdb1334_summary_cope_survey_fields.py]: I don't see the inverse of this statement in `upgrade_rdr()`. Is this supposed to be here?

Comment (kenny-skaggs) [reply rdr_service/alembic/versions/16452fdb1334_summary_cope_survey_fields.py]: Nope. I'm not sure where it came from when I generated the migration, but I'll remove it.

File: rdr_service/config/base_config.json [hunk 2]
```diff
@@ -11,7 +11,10 @@
     "questionnaireOnFamilyHealth",
     "questionnaireOnHealthcareAccess",
     "questionnaireOnMedicalHistory",
-    "questionnaireOnMedications"
+    "questionnaireOnMedications",
+    "questionnaireOnCopeMay",
+    "questionnaireOnCopeJune",
+    "questionnaireOnCopeJuly"
```
Comment (robabram) [inline rdr_service/config/base_config.json:17]: Why are these being added to the app config?

Comment (kenny-skaggs) [reply rdr_service/config/base_config.json:17]: The questionnaire_response_dao uses count_completed_ppi_modules at [line 75](https://github.com/all-of-us/raw-data-repository/blob/devel/rdr_service/dao/questionnaire_response_dao.py#L75) to update numCompletedPPIModules. The ticket seems to be asking for it.

**Review by robabram — APPROVED** [review]

**Review by robabram — APPROVED** [review]

**Pull request merged**